setup.cfg

```diff
@@ -59,6 +59,7 @@ packages =
 addopts =
     --cov tsfresh --cov-report term-missing
     --verbose
+    --benchmark-min-rounds=2
```

Comment:
wouldn't it make more sense to move that option into the workflow benchmark.yml? We are not conducting a benchmark if we run the test suite, are we?